configure.ac

```diff
@@ -2068,6 +2068,8 @@ cts/Makefile					        	\
 	cts/benchmark/Makefile					\
 	cts/benchmark/clubench					\
 	cts/lxc_autogen.sh					\
+	cts/pacemaker-cts-dummyd				\
```

Comment:
This one will need its own line

    AC_CONFIG_FILES([cts/pacemaker-cts-dummyd], [chmod +x cts/pacemaker-cts-dummyd])

otherwise it won't be executable in the source directory